modules/Blockchain.js

```diff
@@ -1035,6 +1037,48 @@ class Blockchain {
         }
         return this.blockchain[0].getIdentity();
     }
+
+    /**
+     * Returns wallet public and private key from configuration
+     */
+    getWallet(blockchain_id) {
+        if (blockchain_id) {
+            const implementation = this.blockchain.find(e => e.getBlockchainId() === blockchain_id);
+
+            if (implementation) {
+                return implementation.getWallet();
+            }
+        }
+        return this.blockchain[0].getWallet();
```

Comment:
We agreed that all methods in blockchain.js should return same JSON structure, so we should update getWallet method to return :
{
blockchain_id:
response:
}
Same comment for get gas price and get trac price

Reply:
We will merge this with Uros PR first